pkg/cmd/admin/admin.go

```diff
@@ -27,15 +27,15 @@ import (
 	exipfailover "github.com/openshift/origin/pkg/cmd/experimental/ipfailover"
 	"github.com/openshift/origin/pkg/cmd/server/admin"
 	"github.com/openshift/origin/pkg/cmd/templates"
+	. "github.com/openshift/origin/pkg/cmd/templates"
```

Comment:
Actually better to use `dedent`. `heredoc` aligns with the smallest indentation present, meaning it can still leave indentation I don't want when doing the unwrap.